format.sh

```diff
@@ -1,14 +1,16 @@
+#!/bin/bash
+
 # C++ & Java
-find -name '*.cpp' -o -name '*.c' -o -name '*.h' -o -name '*.java' | xargs clang-format-13 -i
+find -name '*.cpp' -o -name '*.c' -o -name '*.h' -o -name '*.java' | xargs clang-format -i
```

Comment:
I think we should stick to versioned `clang-format` for reproducibility. If the local script picks up `clang-format-11` and CI uses `clang-format-14`, this could become very confusing.

Reply:
It makes sense, but don't we have the same issue with other tools such as `ktlint`?

Reply:
That's a good point. They are easier to install, but I agree we are consistent here. If you think it's easier, then feel free to drop the version suffix here.